src/transformers/tsx.ts

```diff
@@ -84,5 +103,9 @@ function _trimSizingDslSyntax(s: string) {
 }
 
 function isRule(node: ChildNode): node is Rule {
-  return (node as Rule).selector !== undefined;
+  return node.type === "rule";
```

Comment:
The node includes a type key so I didn't see the need to cast to `Rule` and check for the existence of a selector here.